build/zig/build-0.11.sh

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/bash
+#
+# {{{ CDDL HEADER
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+# }}}
+
+# Copyright 2022 OmniOS Community Edition (OmniOSce) Association.
+
+. ../../lib/build.sh
+
+PROG=zig
+VER=0.11.0
+MAJVER=0
+MINVER=11
```

Comment:
Can we derive these values from `VER`?

Reply:
Done. You might want to check the clang script. I originally based by design off of that, but it doesn't actually extract the `MINVER` correctly.

Reply:
The variable name `MINVER` in the clang build script is a bit misleading as what we want there is `$MAJVER.$MINVER`. We should probably consider changing `MINVER` to `MAJMINVER` to avoid confusion. Sorry for this.